From 8b600a0e5e2b1766d10a518049df02a3d9854388 Mon Sep 17 00:00:00 2001
From: tteras <tteras>
Date: Tue, 18 Jun 2013 05:40:36 +0000
Subject: [PATCH 8/9] From Paul Barker: Remove redundant memset after calloc
 that caused compile failures with gcc 4.8 due to error: argument to 'sizeof'
 in 'memset' call is the same expression as the destination; did you mean to
 dereference.

---
 src/racoon/ipsec_doi.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/racoon/ipsec_doi.c b/src/racoon/ipsec_doi.c
index 5890314..84a4c71 100644
--- a/src/racoon/ipsec_doi.c
+++ b/src/racoon/ipsec_doi.c
@@ -1,4 +1,4 @@
-/*	$NetBSD: ipsec_doi.c,v 1.46 2010/12/14 17:57:31 tteras Exp $	*/
+/*	$NetBSD: ipsec_doi.c,v 1.46.4.1 2013/06/18 05:40:36 tteras Exp $	*/
 
 /* Id: ipsec_doi.c,v 1.55 2006/08/17 09:20:41 vanhu Exp */
 
@@ -1183,7 +1183,6 @@ get_proppair_and_doi_sit(sa, mode, doitype, sittype)
 			"failed to get buffer.\n");
 		goto bad;
 	}
-	memset(pair, 0, sizeof(pair));
 
 	bp = (caddr_t)(sab + 1);
 	tlen = sa->l - sizeof(*sab);
-- 
1.8.4.3

